### PR TITLE
ADD: proper multi-label class weights

### PR DESCRIPTION
**Pull request by madisonmay**

### CODE REVIEW

File: finetune/util/imbalance.py [hunk 1]
```diff
@@ -42,6 +42,18 @@ def compute_class_weights(class_weights, class_counts):
     return class_weights
 
 
+def _compute_ratios(counts, n_total, multilabel=False):
+    computed_ratios = {}
+    max_count = max(counts.values())
+    for class_name, count in counts.items():
+        if multilabel:
+            ratio = (n_total - count) / count
```
Comment (lhz1029) [inline finetune/util/imbalance.py]: it seems like the analogous ratio to the single label example would compare with the max frequency, i.e. 
count_ratio = (n_total - counter) / count
max_ratio = max(all_count_ratios)
ratio = max_ratio / count_ratio

Comment (madisonmay) [reply finetune/util/imbalance.py]: i want to have roughly an even penalty for predicting positive vs. negative on a per label basis -- so I think I want to do this adjustment on a per label basis? i think i also see your logic though -- would be going an additional step to control for each labels contribution to the loss as a whole?

Comment (lhz1029) [reply finetune/util/imbalance.py]: oh I see, why not just n_total / count then?

Comment (madisonmay) [reply finetune/util/imbalance.py]: The idea was that with linear upweighting by this ratio, if you track loss over time you would have had the negative class and the positive class contribute equally.  But we never use linear reweighting anyhow... so... either seems sensical.  I'm setting the negative class's reweighting value to 1 later on.

**Pull request closed without merging**